docs/src/pages/docs/crocks/Const.md

```diff
@@ -71,6 +71,29 @@ resetField(changed)
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Const :: c -> Const c a
+```
+
+The constructor for `Const` is a unary function that takes any type as its
+argument. Once constructed, the value will occupy the far left parameter `c`.
```

Comment:
-- Just a suggestion --

The constructor for `Const` is a unary function that takes any value as its argument an results in a Product type whose underlying left-most value is fixed resulting in an immutable value.